src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-workflows/alerts-classic-vs-workflows.mdx

```diff
@@ -0,0 +1,912 @@
+      <td>
+        Open incident counts of all incident regardless of priority
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `open_violations_count_warning`
+      </td>
+      <td>
+        `$OPEN_VIOLATIONS_COUNT_WARNING`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `N/A`
+      </td>
+      <td>
+        `N/A`
+      </td>
+      <td>
+        Open incident counts of all incident regardless of priority
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `owner`
+      </td>
+      <td>
+        `$EVENT_OWNER`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `{{owner}}`
+      </td>
+      <td>
+        `{{owner}}`
+      </td>
+      <td>
+        There is no matching attribute on the issue level.
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `policy_name`
+      </td>
+      <td>
+        `$POLICY_NAME`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        List
+      </td>
+      <td>
+        `{{accumulations.policyName}}`
+      </td>
+      <td>
+        `{{accumulations.policyName.[0]}}`
+      </td>
+      <td>
+        An issue can have multiple policies
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `policy_url`
+      </td>
+      <td>
+        `$POLICY_URL`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `{{policyUrl}}`
+      </td>
+      <td>
+        `{{#if policyUrl}}"policy_url": {{json policyUrl}},{{/if}}`
+      </td>
+      <td>
+        The URL link to the first policy
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `runbook_url`
+      </td>
+      <td>
+        `$RUNBOOK_URL`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        List
+      </td>
+      <td>
+        `{{accumulations.runbookUrl}}`
+      </td>
+      <td>
+        `{{accumulations.runbookUrl.[0] }}`
+      </td>
+      <td>
+        The first runbook associated to a condition in the issue
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `severity`
+      </td>
+      <td>
+        `$SEVERITY`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        `{{priority}}`
+      </td>
+      <td>
+        `{{#eq HIGH priority}}"WARNING"{{else}}{{json priority}}{{/eq}}`
+      </td>
+      <td>
+        An issue has priority, which can have different values than severity
+      </td>
+    </tr>
+
+
+    <tr>
+      <td>
+        `targets`
+      </td>
+      <td>
+        `$TARGETS`
+      </td>
+      <td>
+        string
+      </td>
+      <td>
+        List
+      </td>
+      <td>
+        `accumulations.targetName`
+      </td>
+      <td>
+        `{{#if accumulations.targetName}}{{json accumulations.targetName.[0]}}{{else if entitiesData.entities}}{{json entitiesData.entities.[0].name}}`
```

Comment:
https://github.com/newrelic/docs-website/pull/10619/files#diff-16a828c0c5ac0b9f7206c72d6068883a3a8c8d416a139cf3462c6c9e4927448cR878-R887 is the full template for targets

Reply:
I think I fixed this @dneth, would you mind taking a look?